CONTRIBUTING.md

```diff
@@ -70,8 +194,24 @@ Guidelines to apply clang-format:
 
 - New files should follow the clang-format style.
 - Old files will be completely re-formatted only if we need to touch several lines/functions/methods of that file. In that case, we suggest to first create a PR just re-formatting the files that will be touched. Later we create another PR with the code changes.
-- If we only need to fix a small portion of a file we do not apply clang-format at all, or we just do it in the code block that we touch.
+- If we only need to fix a small portion of a file then we do not apply clang-format at all, or we just do it in the code block that we touch.
 
 More information about clang:
 - Link to [clang-format](https://clang.llvm.org/docs/ClangFormat.html) tool.
 - Link to the [clang-format option](https://clang.llvm.org/docs/ClangFormatStyleOptions.html).
+
+# 5. Reporting Bugs #
+
+Bugs should be reported by creating Issues on GitHub. However, before reporting a bug first check the Issue list if the bug is already known, and only if you cannot find any previous bug report 
+then create a new Issue. When reporting a bug try describe the problem in as much detail as possible and if the bug is triggered by an input file then attach that file to the GitHub Issue.
+
+# Appendix #
+
+# A.1 The Exiv2 Test System #
+
+TODO: Add stuff here like:
+
+- intro to the test system (what we test and why),
```

Comment:
System tests (= running exiv2 and comparing the output to some provided value) are documented under `tests/writing_tests.md` and `tests/doc.md`.

Unit tests use the Google Test framework, their official [Primer](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md) describes it pretty well.

What do you think should go in here additionally (I am asking because I know the systems and thus don't really see what someone not acquainted with it doesn't)?

Reply:
Here's what I wrote a few years ago (before we had unit tests and the new python tests).  http://dev.exiv2.org/projects/exiv2/wiki/How_do_I_run_the_test_suite_for_Exiv2

Reply:
Oh, I didn't know we had `make teste` and `make testv`. Ops.

Reply:
OK, I will add links to the <tt>tests/writing_tests.md</tt> and <tt>tests/doc.md</tt> docs and remove the Appendix. If there are more stuff that is not covered in those docs (like Robin's wiki page) then <tt>tests/writing_tests.md</tt> and <tt>tests/doc.md</tt> should be updated in another PR right?

Reply:
Robin's Wiki page should be probably included anyway, since it covers the old testsuite under `test` whereas the two files cover the new test suite under `tests`

Reply:
OK, I will add that link too.